dao/interfaces.go

```diff
@@ -24,6 +24,12 @@ type SourceDao interface {
 	BulkMessage(resource util.Resource) (map[string]interface{}, error)
 	FetchAndUpdateBy(resource util.Resource, updateAttributes map[string]interface{}) error
 	ToEventJSON(resource util.Resource) ([]byte, error)
+	// Pause pauses the given source and all its dependant applications. It returns the source with all the dependant
+	// applications preloaded, and an error if the source doesn't exist.
+	Pause(id int64) (*m.Source, error)
+	// Resume resumes the given source and all its dependant applications. It returns the source with all the dependant
+	// applications preloaded, and an error if the source doesn't exist.
+	Resume(id int64) (*m.Source, error)
```

Comment:
I don't know whether word `Resume` is more confusing here as we have event unpause - (and thanks to that it is not consistent) - it is obvious that unpause is opposite action of Pause and in word Resume it is less obvious.

Reply:
+ we have also requests called with `...:/id/unpause`

Reply:
Yeah I can't remember now since it's been so long but I think initially they wanted pause/resume but it changed to pause/unpause on the endpoints.

Reply:
It is true that for the sake of consistency I should have kept it as "Unpause", but since I was only changing the internal naming, I thought it was more clear to use the word "resume", since it seems to be the opposite of pausing something. However I'll open a PR refactoring that.